CMSgov/design-system
# [WNMGDS-712]: Change Alert component default heading to h2

## Changed
- Change Alert component default heading to h2

Fixes #880

## How to test
- Run the design system site locally 
- Go to the alerts page and view the source to see that an `<h2>` is used for the alert header when a specific header is not specified
